src/modules/shmem/consumer_shmem.yml

```diff
@@ -0,0 +1,34 @@
+schema_version: 0.2
+type: consumer
+identifier: shmem
+title: Shared memory output
+version: 3
+copyright: Copyright (C) 2016 Meltytech, LLC
```

Comment:
How about moving the code to the "plus" module? It doesn't really have any  external dependencies (except Linux).